active/NavigationView/HierarchicalNav/HierarchicalNavigationView.md

```diff
@@ -0,0 +1,363 @@
+We believe two levels is ideal for usability and comprehension.
+
+NavigationView shows hierarchy in all its pane display modes, including Top and LeftCompact.
+
+*Note:* The following is a comp and will be updated with screenshots before this doc is published.
+![NavigationView in Left, LeftCompact, and Top modes showing hierarchy](NavigationView_Hierarchy.png)
+
+# Examples
+
+## Add a hierarchy of items in markup
+Declare app navigation hierarchy in markup.
+
+```Xaml
+<muxc:NavigationView>
+    <muxc:NavigationView.MenuItems>
+        <muxc:NavigationViewItem Content="Home" Icon="Home" ToolTipService.ToolTip="Home"/>
+        <muxc:NavigationViewItem Content="Collections" Icon="Keyboard" ToolTipService.ToolTip="Collections">
+            <muxc:NavigationViewItem.MenuItems>
+                <muxc:NavigationViewItem Content="Bookshelf" Icon="Library" 
+                    ToolTipService.ToolTip="Bookshelf"/>
+                <muxc:NavigationViewItem Content="Mail" Icon="Mail" ToolTipService.ToolTip="Mail"/>
+            </muxc:NavigationViewItem.MenuItems>
+        </muxc:NavigationViewItem>
+    </muxc:NavigationView.MenuItems>
+</muxc:NavigationView>
+```
+
+## Add a hierarchy of items using data binding
+
+Add a hierarchy of menu items to the NavigationView by 
+* binding the MenuItemsSource property to the hierarchical data
+* defining the item template to be a NavigationViewMenuItem, with its Content set to be the label of the menu item, and its its MenuItemsSource property bound to the next level of the hierarchy
+
+This example also demonstrates the Expanding and Collapsing events. These events are raised for a menu item with children.
+
+```xaml
+<DataTemplate x:Key="NavigationViewMenuItem" x:DataType="local:Category">
+    <muxc:NavigationViewItem Content="{x:Bind Name}" MenuItemsSource="{x:Bind Children}"/>
+</DataTemplate>
+
+<muxc:NavigationView x:Name="navview" 
+    MenuItemsSource="{x:Bind categories, Mode=OneWay}" 
+    MenuItemTemplate="{StaticResource NavigationViewMenuItem}" 
+    ItemInvoked="{x:Bind OnItemInvoked}" 
+    Expanding="OnItemExpanding" 
+    Collapsed="OnItemCollapsed" 
+    PaneDisplayMode="Left">
+    
+    <StackPanel Margin="10,10,0,0">
+        <TextBlock Margin="0,10,0,0" x:Name="ExpandingItemLabel" Text="Last Expanding: N/A"/>
+        <TextBlock x:Name="CollapsedItemLabel" Text="Last Collapsed: N/A"/>
+    </StackPanel>    
+</muxc:NavigationView>
+```
+
+```c#
+public class Category
+{
+    public String Name { get; set; }
+    public String Icon { get; set; }
+    public ObservableCollection<Category> Children { get; set; }
+}
+    
+public sealed partial class HierarchicalNavigationViewDataBinding : Page
+{
+    public HierarchicalNavigationViewDataBinding()
+    {
+        this.InitializeComponent();
+           
+        public ObservableCollection<Category> Categories = new ObservableCollection<Category>(){
```

Comment:
Does this sample compile? Looks like it needs a `}` before this declaration.

Reply:
Recommend: Add a close brace (to end the constructor). And standardize on curly braces at the start of the next line.